crates/forge/src/test_case_summary.rs

```diff
@@ -23,6 +24,8 @@ pub enum TestCaseSummary {
         arguments: Vec<Felt252>,
         /// Statistic for fuzzing test
         fuzzing_statistic: Option<FuzzingStatistics>,
+        /// Number of block used if BlockId::Tag(Latest) was specified
+        block_number_of_latest: Option<BlockNumber>,
```

Comment:
I think we should follow the suit of `FuzzingStatistics` and introduce a struct for a forking parameters (e.g. name it `ForkConfigUsed`) and store this details there.

Reply:
Perhaps we could always store the block number and have a separate boolean flag for if `latest` block was used. Could make the logic simpler.

Reply:
Out of scope: #1049

Reply:
Agree, but imo `Option` expresses it better for now

Reply:
I would leave it for now and wrap it in a struct when additional info will be needed to not introduce unnecessary abstraction layer